packages/docs-site/src/content/docs/guides/node-operators/deploy-a-proverset.mdx

```diff
@@ -43,37 +43,26 @@ This guide outlines how to achieve this with separate EOAs running prover and pr
         </TabItem>
     </Tabs>
 
-2. **Deploy the ProverSet implementation**
+2. **Deploy the ProverSet**
 
-    Run the `DeployProverSetUtil.s.sol` script with your **proposer's private key**. You can find the script [here](https://github.com/taikoxyz/taiko-mono/tree/main/packages/protocol/script/DeployProverSetUtil.s.sol).
-    
-    ```bash
-    forge script --chain-id {CHAIN_ID} --broadcast --rpc-url {YOUR_RPC_URL} --private-key=$PRIVATE_KEY script/DeployProverSetUtil.s.sol:DeployProverSetUtil
-    ```
-
-    The script should print your implementation address with the log:
-
-    `New impl address is: 0x....`
-
-    You will need the address for the next step.
+    Set `ROLLUP_ADDRESS_MANAGER` to the address of the RollupAddressManager contract on the network you are deploying to. You can find these values in our network reference docs.
 
-3. **Deploy the Proxy and init the ProverSet smart contract**
-
-    Fill in the implementation address, set the **prover** EOA as admin, and ensure that the `ROLLUP_ADDRESS_MANAGER` matches for the respective network. You can find these values in our network reference docs.
-
-    Run the `DeployProverSet.s.sol` script with your **proposer's private key** , altering the values in the script as necessary. You can find the script [here](https://github.com/taikoxyz/taiko-mono/tree/main/packages/protocol/script/DeployProverSet.s.sol).
+    Run the `DeployProverSet.s.sol` script with your **proposer's private key**. You can find the script [here](https://github.com/taikoxyz/taiko-mono/tree/main/packages/protocol/script/DeployProverSet.s.sol).
 
     ```bash
```

Comment:
Maybe include a prompt to set `PROVER_SET_ADMIN` to the prover EOA for people who don't go and read the script? Otherwise will automatically set to the msg.sender which should be proposer EOA